Tests/reviews.api.test.js

```diff
@@ -1,124 +1,165 @@
-      });
-
-  });
-
-
-  test("/Reviews/getReviewSummary/:productId : it should respond with an object, respond 200 for valid response ", async () => {
-    var productId = 1000;
-    var mockData = await SeedData.generateSeedData(10, 1);//load 10 customer,  for 1 product
-    await Service.insertSeedData(mockData);
-
-    return request(app)
-      .get(`/Reviews/getReviewSummary/${productId}`)
-      .then(response => {
-        expect(response.statusCode).toBe(200);
-        expect(Array.isArray(response.body)).toEqual(false);
-        expect(typeof (response.body)).toEqual('object');
-        var insertedData = response.body;
-        expect.objectContaining({
-          averageRating: expect.any(String),
-          totalRatings: expect.any(Number),
-          fiveStar: expect.any(String),
-          fourStar: expect.any(String),
-          threeStar: expect.any(String),
-          twoStar: expect.any(String),
-          oneStar: expect.any(String)
-        });
-      });
-
-  });
+});
 
-  test("/Reviews/getReviewExcerpts/:productId : it should respond with an array, respond 200 for valid response ", async () => {
-    var productId = 1000;
-    var mockData = await SeedData.generateSeedData(50, 1);//load 10 customer,  for 1 product
-    await Service.insertSeedData(mockData);
+describe('/review', () => {
 
-    return request(app)
-      .get(`/Reviews/getReviewExcerpts/${productId}`)
-      .then(response => {
-        expect(response.statusCode).toBe(200);
-        expect(Array.isArray(response.body)).toEqual(true);
-        var insertedData = response.body;
-        expect(insertedData.length).toBeLessThanOrEqual(12);
-      });
+  const db = require('../database-mongoose/reviews.service');
+  db.createReview.mockImplementation(() => 'created');
 
+  test('POST request returns result of createReview', async (done) => {
+    const response = await request.post('/review');
+    expect(response.text).toEqual('created');
+    done();
   });
 
-  test("/Reviews/getReviewsByFeature/:productId : it should respond with an object, respond 200 for valid response ", async () => {
-    var productId = 1000;
-    var mockData = await SeedData.generateSeedData(10, 1);//load 10 customer,  for 1 product
-    await Service.insertSeedData(mockData);
-
-    return request(app)
-      .get(`/Reviews/getReviewSummary/${productId}`)
-      .then(response => {
-        expect(response.statusCode).toBe(200);
-        expect(Array.isArray(response.body)).toEqual(false);
-        expect(typeof (response.body)).toEqual('object');
-        var insertedData = response.body;
-        expect.objectContaining({
-          easeToUse: expect.any(String),
-          voiceRecognition: expect.any(String),
-          techSupport: expect.any(String),
-          valueForMoney: expect.any(String),
-          qualityOfMaterial: expect.any(String),
-          batteryLife: expect.any(String)
-        });
-      });
-
-  });
+});
 
+// describe('test the root path', () => {
+
```

Comment:
if you use `xdescribe(..`  or `xtest(...` jest will skip the bloc without running but keeps them in the summary